### ISSUE TITLE
onComplete event does not dump anything on the screen

### ISSUE THREAD

**Issue by Rufflesaurus** [masihyeganeh/YoutubeDownloader#12]
I wanna do something when the video is completely downloaded but the onComplete event seems like it's not working because when I have something like this:

```
       $youtubeDownloader = new YoutubeDownloader('qPwX1gkPRic');

        $youtubeDownloader->setPath('youtube-downloads');

        $youtubeDownloader->onComplete = function ($filePath, $fileSize, $index, $count) {
            var_dump('Download is complete!');
        };

        $youtubeDownloader->download();
```

I don't see anything on the screen.

The onProgress event works fine.

**Comment by masihyeganeh** [comment]
I think I forgot to add `$index` and `$count` parameters to onComplete function.
Can you please check this code and let me know if it works or not
```php
<?php
       $youtubeDownloader = new YoutubeDownloader('qPwX1gkPRic');

        $youtubeDownloader->setPath('youtube-downloads');

        $youtubeDownloader->onComplete = function ($filePath, $fileSize) {
            var_dump('Download is complete!');
        };

        $youtubeDownloader->download();
```

Will fix this in next release soon.

**Comment by Rufflesaurus** [comment]
Yup, that worked!

**Comment by masihyeganeh** [comment]
Good. I fixed it in v2.1. This function now has 4 parameters. please consider this if you are upgrading
